keyboards/handwired/ortho5x14/rules.mk

```diff
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no       # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = yes        # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+TAP_DANCE_ENABLE = yes
```

Comment:
Tap dance should be supported at the keymap level, not the keyboard level. 
```suggestion
```

Reply:
How do I turn on Tab Dance at the keymap level?  All documentation (see: https://docs.qmk.fm/#/feature_tap_dance) and examples say line #22 is the way to go.  Can you provide an example of what to do to re-enable Tap Dance at the keymap level?

Reply:
Just add a rules.mk in the keymap directory with:
```make
TAP_DANCE_ENABLE = yes
```

Reply:
Wow, that was easy.  Done.